src/cc/qfsc/test-qfsc.c

```diff
@@ -339,6 +339,50 @@ static char* test_qfs_get_data_locations() {
   return 0;
 }
 
+// Append to a file by Writing, closing, seeking, and then writing again. This
+// emulates a typical unix O_APPEND operation without using the flag.
+//
+// Using O_APPEND itself does not produce the same result.  Opening a file with
+// O_APPEND and writing to it will create hole of the appropriate size to make
+// the previous chunk reach the chunksize.  This can be useful as an atomic
```

Comment:
"This can be useful as an atomic append, but is not the behavior you'd expect from a normal filesystem."

I don't think this statement is needed here.